### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by narcis96**
# Description
Some of the commands in the README were not up to date.

### CODE REVIEW

**Review by MartinquaXD — COMMENTED** [review]
Could you turn this into a general "update readme" PR and also drop the whole section about setting up a node and postgres + migrations on your host system without docker?
Unless someone in the team strongly prefers to keep that I would like to move towards using docker for all those things to make it easier for everybody involved.

**Comment by narcis96** [comment]
> Could you turn this into a general "update readme" PR and also drop the whole section about setting up a node and postgres + migrations on your host system without docker? Unless someone in the team strongly prefers to keep that I would like to move towards using docker for all those things to make it easier for everybody involved.

done

File: README.md [hunk 1]
```diff
@@ -62,9 +62,9 @@ The CI runs unit tests, e2e tests, `clippy` and `cargo fmt`
 
 `cargo test`
 
-### Integration Tests:
+### DB Tests:
```
Comment (fleupold) [inline README.md:65]: nit: We do still have integration tests (and e2e tests) don't we (should be running as part of CI). Maybe it's enough to link the workflow file but would be nice to include somehow.

File: README.md [hunk 2]
```diff
@@ -88,63 +88,10 @@ The CI runs unit tests, e2e tests, `clippy` and `cargo fmt`
...
-In case you run into `java.net.UnknownHostException: host.docker.internal` add `--add-host=host.docker.internal:host-gateway` right after `docker run`.
-
-If you're combining a local postgres installation with docker flyway you have to add to the above `--network host` and change `host.docker.internal` to `localhost`.
-
-- Local [flyway installation](https://flywaydb.org/documentation/usage/commandline/#download-and-installation)
+The tests that require postgres connect to the default database of a locally running postgres instance on the default port. Running the command below, the migrations will be applied as well:
 
 ```sh
-flyway -user=$USER -password="" -locations="filesystem:database/sql/" -url=jdbc:postgresql:/// migrate
+docker-compose up -d
```
Comment (fleupold) [inline README.md:94]: nit: Isn't it annoying to run in detached mode (not trivial to kill)? Maybe mention to run in a separate shell?

**Review by fleupold — APPROVED** [review]
Noice

File: README.md (hunk 2 above)
Comment (narcis96) [reply README.md]: done

**Pull request merged**